### ISSUE TITLE
Backslash is not unescaped

### ISSUE THREAD

**Issue by vibin** [jnbt/java-properties#13]
For `hello=wo\\d`, the API `JavaProperties.load("sample.properties")['hello'.to_sym]` returns `wo\\d` instead of `wo\d`.

**Comment by vibin** [comment]
TBF, I'm not sure if this is a bug (i'm new to ruby), but the JDK Properties API definitely does unescape backslashes.

**Comment by jnbt** [comment]
@vibin I added a fix for this behavior. Can you test the new version? If you use this gem via a Bundler you can simply use the following line to your `Gemfile` load the unpublished version:

```
gem 'java-properties', github: 'jnbt/java-properties', branch: 'fix-backslash-not-unescaped'
```

and then run `bundle install`

**Comment by vibin** [comment]
@jnbt this fixes the issue. Thank you!

**Comment by jnbt** [comment]
@vibin Thanks for your feedback. The fix has been released as version `0.2.1`: https://rubygems.org/gems/java-properties/versions/0.2.1
